src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1540,6 +1551,42 @@ impl Coordinator {
                         role_revokes.insert((*group_id, *id, *grantor_id));
                     }
                 }
+                ObjectId::Item(id) => {
+                    if let Some(index) = self.catalog().get_entry(id).index() {
+                        let humanizer = self.catalog().for_session(session);
+                        let dependants = self
+                            .controller
+                            .compute
+                            .reverse_compute_dependencies(*id, index.cluster_id)
+                            .map(|it| it.collect_vec())
+                            .unwrap_or(Vec::<&GlobalId>::new())
+                            .into_iter()
```

Comment:
You can avoid the `Vec::new()` allocations

```suggestion
                            .reverse_compute_dependencies(*id, index.cluster_id)
                            .ok()
                            .into_iter()
                            .flatten()
```

Reply:
Thanks, done